postcss.config.js

```diff
@@ -6,13 +6,13 @@ const { cosmiconfigSync } = require('cosmiconfig')
 
 const cssModulesConfig = cosmiconfigSync('cssmodules').search().config
 
-module.exports = {
+module.exports = ({ webpack }) => ({
   map: true,
   plugins: [
     presetEnv(),
-    cssModules({
+    webpack && cssModules({
       ...cssModulesConfig,
       getJSON: () => {}
     }),
-  ],
-}
+  ].filter(x => x),
```

Comment:
To remove one `false` because as far as I know Babel does a check for being actual plugin name and throws error. I am more really sure about. Should I remove it?